helm/templates/deployment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "feedback.fullname" . }}
+  labels:
+{{ include "feedback.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "feedback.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "feedback.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "feedback.fullname" . }}
+          env:
+            - name: GO_ENV
+              value: production
+            - name: EMAIL_NOREPLY
+              value: feedback@pubg.com
+            - name: HTTP_PORT
+              value: 80
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 7000
+              protocol: TCP
+          volumeMounts:
+            - mountPath: /app/.env
+              name: env-secrets
+              subPath: .env
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
```

Comment:
I think this service doesn't actually have healthcheck endpoints ... maybe just getting a 200 from `/` is ok...